scim.sgnl.yaml

```
@@ -0,0 +1,456 @@
+description: "SCIM 2.0 as a System of Record"
+address: "{{Input Required: SCIM URL}}"
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: "SCIM2.0-1.0.0"
+# Example Configs:
+# empty config: {} | b64: "e30="
+# config with filtering on Users: {"Users":{"queryParams":{"filter":"displayName eq \"SGNL\""}}} | b64: "eyJVc2VycyI6eyJxdWVyeVBhcmFtcyI6eyJmaWx0ZXIiOiJkaXNwbGF5TmFtZSBlcSBcIlNHTkxcIiJ9fQ=="
+# config with filter, ordering and sortBy on Groups {"Groups":{"queryParams":{"filter":"displayName eq \"SGNL\"","sortBy":"userName","ascending":true}}} | b64: "eyJHcm91cHMiOnsicXVlcnlQYXJhbXMiOnsiZmlsdGVyIjoiZGlzcGxheU5hbWUgZXEgXCJTR05MXCIiLCJzb3J0QnkiOiJ1c2VyTmFtZSIsImFzY2VuZGluZyI6dHJ1ZX19fQ=="
+adapterConfig: "e30=" # {}
+# This must match the list of supportedAuthMechanisms specified on the Adapter.
+auth:
+  - basic:
+      username: "{{Input Required: Username}}"
+      password: "{{Input Required: API Token}}"
+  - bearer:
+      authToken: "{{Input Required: token}}"
+  - oAuth2ClientCredentials:
+      clientId: "{{Input Required: Client-ID}}"
+      clientSecret: "{{Input Required: Client-Secret}}"
+      tokenUrl: "{{Input Required: AuthTokenURL}}"
+      scope: "{{Input Required: Scope}}"
+      authStyle: AutoDetect
+entities:
+  User:
+    displayName: SCIMUser
+    externalId: Users
+    description: User entity in SCIM
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: externalId
+        externalId: externalId
+        type: String
```

Might be good to index.
```suggestion
      - name: externalId
        externalId: externalId
        type: String
        indexed: true
```